src/index.ts

```diff
@@ -111,6 +111,7 @@ export { ReturnDocument } from './operations/find_and_modify';
 export { ProfilingLevel } from './operations/set_profiling_level';
 export { ReadConcernLevel } from './read_concern';
 export { ReadPreferenceMode } from './read_preference';
+export { AsyncDisposable } from './resource_management';
```

Comment:
```suggestion
export { type AsyncDisposable } from './resource_management';
```

Also, minor, this is grouped with the enums, but that's something we've manually kept organized.

Reply:
What is the value of organization of this file?  This frequently comes up in reviews and it's primarily just you requesting changes for organization purposes.  But best I can tell, there's no impact for us or users if it's not organized

Reply:
No need to change it, that's why I call out that it has been manually maintained, it does provide visual organization for us when reading this file.

The important part here is the `type` export